doc/fileformat/serdes.rst

```diff
@@ -0,0 +1,262 @@
+.. _formatserdes:
+
+Checkpointing
+=============
+
+While not a fileformat on its own, checkpoints allow users to dump a snapshot of
+the simulation state for a later time. This is **not** intended for data
+extraction (see :ref:`probesample` for this functionality), but instead a reset
+the simulation state to this point in time. The saved data is not a complete
+image of the state, but strives to be a minimal subset from which to restore
+(see :ref:`below <impl guide>`).
+
+Checkpoints are currently _not_ portable; i.e. the following constraints apply:
```

Comment:
```suggestion
Checkpoints are currently **not** portable; i.e. the following constraints apply:
```

Reply:
Why favour **not** over _not_ here?

Reply:
Still, one is bold, the other italic?

Reply:
Bold is to be consistent with the previous emphasized not.